File: PythagoreanNumbers/PythagoreanNumbers.java
import java.util.Scanner;

public class PythagoreanNumbers {
	
	public static void main(String[] args) {
		
		Scanner scan = new Scanner(System.in);
		
		int a = scan.nextInt();
		int b = scan.nextInt();
		int c = scan.nextInt();
		
		System.out.println(
			(
				a * a == b * b + c * c ||
				b * b == a * a + c * c ||
				c * c == a * a + b * b
			) ? "YES" : "NO"
		);
	}
}

File: PythagoreanNumbers/PythagoreanNumbers.py
a = int(input())
b = int(input())
c = int(input())
if a*a == b*b + c*c:
    print("YES")
elif b*b == c*c + a*a:
    print("YES")
elif c*c == a*a + b*b:
    print("YES")
else:
    print("NO")
